packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1404,18 +1446,37 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | TeamsCa
     this.emitter.emit('transferAccepted', args);
   }
 
-  private capabilitiesChanged(data: CapabilitiesChangeInfo): void {
+  private capabilitiesChangedturnVideoOnTrampoline(data: CapabilitiesChangeInfo): void {
+    /* @conditional-compile-remove(media-access) */
+    if (data.newValue.turnVideoOn?.isPresent === false) {
+      this.disposeLocalVideoStreamView();
```

Comment:
this should still be `stopCamera` as that handler will trigger other state changes needed for the UI

Reply:
The stop camera should be called in web sdk, not from UI library, I fixed it in sdk.